doc/docker.md

```diff
@@ -4,6 +4,12 @@ Docker is well known among software developers. Even if you are not familiar wit
 
 ## Build openfang using pre-built docker image (recommended)
 
+! Some users have problems compiling Openfang with GCC 2.28 and M4 (Ubuntu 18 or later ). We recomend Ubuntu 16.04.6 LTS to deploy Openfang docker. Make sure that you have the following packages in the system:
```

Comment:
I couldn't understand this note! Please remove it or turn it more explicit. Note that the docker image contains the available toolchain.